File: app/src/main/java/com/akhbulatov/githubsample/models/UserDetails.java
package com.akhbulatov.githubsample.models;

import com.google.gson.annotations.SerializedName;

public final class UserDetails {
    @SerializedName("id")
    public long id;

    @SerializedName("login")
    public String login;

    @SerializedName("avatar_url")
    public String avatarUrl;

    @SerializedName("name")
    public String name;

    @SerializedName("location")
    public String location;

    @SerializedName("public_repos")
    public int publicRepos;

    @SerializedName("followers")
    public int followers;

    @SerializedName("following")
    public int following;
}

File: app/src/main/java/com/akhbulatov/archsample/models/UserDetails.kt
package com.akhbulatov.archsample.models

import com.google.gson.annotations.SerializedName

data class UserDetails(
    @SerializedName("id") val id: Long,
    @SerializedName("login") val login: String,
    @SerializedName("avatar_url") val avatarUrl: String,
    @SerializedName("name") val name: String,
    @SerializedName("location") val location: String,
    @SerializedName("public_repos") val publicRepos: Int,
    @SerializedName("followers") val followers: Int,
    @SerializedName("following") val following: Int
)